### DISCUSSION TITLE
Accept script-specific parameters from commandline

### DISCUSSION THREAD

**Discussion by WampyCakes** [ManimCommunity/manim#2786] [Ideas]
## Description of proposed feature
There ought to be a good way to pass parameters that are not manim flags to a script. I imagine this is its own "section" in the command where these would belong. Maybe after the script and scene name (explicit or implied for scene name)?

## How can the new feature be used?
Two immediate use cases that come to mind are:
- Things like generative art or similar concepts where it's ideal to tweak a few parameters in the command to change output
- https://github.com/WampyCakes/manim-rubikscube#putting-it-all-together here I benefited from passing a cube state through the commandline, but manim throws a warning with this approach, making it not an ideal solution.

## Additional comments
This should not be worked on until after #1013 is merged.

**Comment by WampyCakes** [comment]
[This may be important per jsonV.](https://click.palletsprojects.com/en/7.x/advanced/?highlight=usage#forwarding-unknown-options)

**Comment by MrDiver** [comment]
- https://github.com/ManimCommunity/manim/pull/1013
Is now merged

Related to
- #1888

Please discuss further implementation of this here and compatibility with the new version of manim
